ads/google/a4a/docs/a4a-readme.md

```diff
@@ -1,30 +1,30 @@
-# A4A - AMP for Ads
+# AMP Ads
 
-A4A applies  AMP’s core philosophy of reliable fast performance and  great user experience to ads. 
+AMP Ads applies AMP’s core philosophy of reliable fast performance and  great user experience to ads. 
 
 # AMP Ads 
 
-AMP ads are written in AMP format - [A4A HTML](https://github.com/google/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) (A variant of AMP HTML) + CSS. This means that ads can no longer have the ability to run arbitrary JavaScript - which is traditionally the number one cause of poor ad performance. Therefore, just like core AMP, the core ads JavaScript use-cases are built right into the AMP Open Source project which guarantees good behavior from ads. 
+AMP ads are written in AMP format - [AMP Ads HTML](https://github.com/google/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) (A variant of AMP HTML) + CSS. This means that ads can no longer have the ability to run arbitrary JavaScript - which is traditionally the number one cause of poor ad performance. Therefore, just like core AMP, the core ads JavaScript use-cases are built right into the AMP Open Source project which guarantees good behavior from ads. 
```

Comment:
@gbender03 Worth fixing "ads" on this line too. You can leave the A4A HTML as is because the format spec is still referred as A4A.